Pb_Info/Clasa_9/Sortarea_vectorilor/Pagina2/Sortare/Sortare.cpp

```diff
@@ -0,0 +1,34 @@
+#include<iostream>
+#include<fstream>
+
+using namespace std;
+
+ifstream intr ("fis.in");
+
+int main()
+{
+    int x[1000],a;
+    intr>>a;
+    for(int i=0;i<a;i++){
+        intr>>x[i];
+    }
+
+    for(int i=0;i<a;i++){
+        int p=i;
+        for(int j=i;j<a;j++){
+            if(x[j]<x[i]){
+                p=j;
+            }
+        }
+        int max=x[i];
+        x[i]=x[p];
+        x[p]=max;
+    }
+    for(int i=a-1;i>=0;i--){
+        cout<<x[i]<<" ";
+    }
+
+
+
```

Comment:
Program does not sort the vector properly. Check your solution again

Reply:
Rezolva si pagina 2 de la sortarea vectorilor. (toata)
https://www.pbinfo.ro/probleme/categorii/3/tablouri-unidimensionale-vectori-sortarea-vectorilor?start=10